sqlx-macros/src/derives/attributes.rs

```diff
@@ -72,6 +73,7 @@ pub fn parse_container_attributes(input: &[Attribute]) -> syn::Result<SqlxContai
                                     "lowercase" => RenameAll::LowerCase,
                                     "snake_case" => RenameAll::SnakeCase,
                                     "uppercase" => RenameAll::UpperCase,
+                                    "screaming_snake_case" => RenameAll::ScreamingSnakeCase,
```

Comment:
I wasn't sure if we want the text name to more closely match/mimic the desired case (eg `"SCREAMING_SNAKE_CASE"`). Let me know what you think 🙂

Reply:
I think we should match the convention of [`#[serde(rename_all = "...")]`](https://serde.rs/container-attrs.html#rename_all) since it's pretty clear that is what this feature is mimicking.

So yeah `"uppercase"` should be `"UPPERCASE"` and this should be `"SCREAMING_SNAKE_CASE"`. Since we didn't have `"uppercase"` yet in 0.3.5 this should be fine to change.

Reply:
Sorry for the delay (busy week last week), I've gone ahead and updated the names here to more closely match the styles they're for, and match serde!